### DISCUSSION TITLE
Cannot read properties of null (reading 'map')

### DISCUSSION THREAD

**Discussion by Jam0r85** [GuavaCZ/calendar#41] [Q&A] [answered]
I'm getting an issue with this which I cannot figure out.

When following the demo package and setting up my CalendarWidget the calendar container displays however I don't get the calendar itself to display. Please see below. Can anyone shed any light on this as I cannot figure out what is causing this.

![Screenshot 2024-10-16 at 15 14 21](https://github.com/user-attachments/assets/2d1aea1d-c2f9-4a47-9d6e-182f0128f3fc)
![Screenshot 2024-10-16 at 15 14 31](https://github.com/user-attachments/assets/0a6af635-bdaa-43ce-ad10-4a6c6478b144)
![Screenshot 2024-10-16 at 15 14 54](https://github.com/user-attachments/assets/2121ead7-75e5-471e-86b3-242216c4d145)
![Screenshot 2024-10-16 at 15 15 12](https://github.com/user-attachments/assets/0dc80273-f3a7-4871-b934-bacbbd298133)

**Comment by lukas-frey** [comment]
DId you follow every step here? https://github.com/GuavaCZ/calendar?tab=readme-ov-file#installation

The demo repo is not entirely up to date, so best is to check the docs and follow the instructions there.

**Comment by Jam0r85** [comment, marked as answer]
Managed to solve this.

Even though I don't plan on using resources I had to create a 'fake' resource in my widget and it loaded up first time.

![Screenshot 2024-10-17 at 09 47 54](https://github.com/user-attachments/assets/13312dbb-8e8f-4096-aefb-312c29cb8bec)

**Reply by lukas-frey** [reply]
That doesn't sound right. 🤔 Are you using resource views? Just use a non-resource view if you don't need resources.

**Reply by Jam0r85** [reply]
Nope, default view is timeGridWeek and in the header I have dayGridMonth,timeGridWeek,timeGridDay,listWeek?

I've deleted that function to see and I get the same JS error when trying to load the calendar. The calendar only loads up with that function in place?
